packages/components/src/components/link/link.test.stories.ts

```diff
@@ -0,0 +1,347 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import '../../solid-components';
+import { html } from 'lit-html';
+import {
+  storybookDefaults,
+  storybookHelpers,
+  storybookTemplate,
+  storybookUtilities
+} from '../../../scripts/storybook/helper';
+import { userEvent } from '@storybook/test';
+import { waitUntil } from '@open-wc/testing-helpers';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+const { argTypes } = storybookDefaults('sd-link');
+const { generateTemplate } = storybookTemplate('sd-link');
+const { overrideArgs } = storybookHelpers('sd-link');
+const { generateScreenshotStory } = storybookUtilities;
+
+export default {
+  title: 'Components/sd-link/Screenshot Tests',
+  component: 'sd-link',
+  tags: ['!autodocs'],
+  args: overrideArgs([
+    { type: 'slot', name: 'default', value: 'Link' },
+    { type: 'attribute', name: 'href', value: '#' }
+  ]),
+  argTypes,
+  decorators: [withActions] as any
+};
+
+/**
+ * Default: This shows sd-link in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
+
+/**
+ * Unset `href` to make the link disabled.
+ */
+
+export const InvertedAndDisabled = {
+  name: 'Disabled × Inverted',
+  parameters: { controls: { exclude: ['size', 'default', 'href', 'inverted'] } },
+  render: (args: any) =>
+    generateTemplate({
+      axis: {
+        x: {
+          type: 'attribute',
+          name: 'href',
+          values: ['#link', { value: '', title: '–' }]
+        },
+        y: {
+          type: 'attribute',
+          name: 'inverted',
+          values: [false, true]
+        }
+      },
+      args,
+      options: {
+        templateBackgrounds: { alternate: 'y', colors: ['white', 'rgb(var(--sd-color-primary, 0 53 142))'] }
+      }
+    })
+};
+
+/**
+ * You can make links bold by setting `<b>` tags around the text in the main slot.
+ */
+
+export const BoldInMainSlot = {
+  name: 'Bold',
+  parameters: { controls: { exclude: ['default'] } },
+  render: (args: any) =>
+    generateTemplate({
+      axis: {
+        x: {
+          type: 'slot',
+          name: 'default',
+          title: 'main-slot',
+          values: ['Link', '<b>Link</b>']
+        }
+      },
+      args,
+      options: {
+        templateBackgrounds: { alternate: 'y', colors: ['white', 'rgb(var(--sd-color-primary, 0 53 142))'] }
+      }
+    })
+};
+
+/**
+ * Use the `icon-left` and `icon-right` slots to add icons. They automatically adapt the size.
```

Comment:
* Use the `icon-left` and `icon-right` slots to add system icons. They automatically adapt the size.